pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -127,6 +127,29 @@ rec {
     };
   };
 
+  bytecode-outline = buildEclipsePlugin rec {
+    name = "bytecode-outline-${version}";
+    version = "2.4.3";
+
+    srcFeature = fetchurl {
+      url = "http://andrei.gmxhome.de/eclipse/features/de.loskutov.BytecodeOutline.feature_${version}.jar";
+      sha256 = "0imhwp73gxy1y5d5gpjgd05ywn0xg3vqc5980wcx3fd51g4ifc67";
+    };
+
+    srcPlugin = fetchurl {
+      url = "http://dl.bintray.com/iloveeclipse/plugins/de.loskutov.BytecodeOutline_${version}.jar";
```

Comment:
One is the "feature jar" and the other the actual "plugin jar". For whatever reason upstream has decided to host these on different sites. Maybe because the plugin jar is quite a lot larger? In any case these are the official download URLs according to the Eclipse update site: http://andrei.gmxhome.de/eclipse/site.xml